var/spack/repos/builtin/packages/kbd/package.py

```diff
@@ -0,0 +1,25 @@
+# Copyright 2013-2020 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class Kbd(AutotoolsPackage):
+    """The kbd project contains tools for managing Linux console
+    (Linux console, virtual terminals, keyboard, etc.) – mainly,
+    what they do is loading console fonts and keyboard maps."""
```

Comment:
You're getting a syntax error in here somewhere. Maybe some kind of unicode slipped in? Try replacing - with -.

Reply:
Got it, fixed in https://github.com/spack/spack/pull/18436/commits/2ebaca6dbe01448e59341da97d3d5c278152f3db